### PR TITLE
Trim requirements

### PR DESCRIPTION
**Pull request by kragniz**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,40 +3,25 @@
...
 
-# < 0.8.0/0.8 does not work, see https://bugs.launchpad.net/bugs/1153983
-SQLAlchemy>=0.9.7,<=0.9.99
-anyjson>=0.3.3
 eventlet>=0.17.3
 PasteDeploy>=1.5.0
 Routes>=1.12.3,!=2.0
 WebOb>=1.2.3
-sqlalchemy-migrate>=0.9.5
 httplib2>=0.7.5
```
Comment (sigmavirus24) [inline requirements.txt:16]: Probably don't need this.

File: requirements.txt [hunk 2]
```diff
@@ -3,40 +3,25 @@
...
-kombu>=3.0.7
 pycrypto>=2.6
```
Comment (sigmavirus24) [inline requirements.txt:17]: Shouldn't need this either.

File: requirements.txt [hunk 3]
```diff
@@ -47,21 +32,13 @@ pyOpenSSL>=0.11
 # Required by openstack.common libraries
 six>=1.9.0
 
-oslo.db>=1.7.0  # Apache-2.0
 oslo.i18n>=1.5.0  # Apache-2.0
 oslo.log>=1.0.0  # Apache-2.0
 oslo.messaging>=1.8.0  # Apache-2.0
```
Comment (sigmavirus24) [inline requirements.txt:37]: Will this API be publishing events? If not, do we still need oslo.messaging?

File: requirements.txt (hunk 1 above)
Comment (kragniz) [inline requirements.txt:16]: It's currently required for keystone auth in searchlight/common/auth.py, but that custom code should be ripped out

File: requirements.txt (hunk 3 above)
Comment (sjmc7) [inline requirements.txt:37]: We're consuming notifications for indexing; oslo-messaging might add some cruft we don't need but i think we'll keep it until it's easier not to.

Comment (sjmc7) [inline requirements.txt:37]: Ahh... osprofiler requires a notifier to be configured. So I think we do need to keep oslo-messaging and kombu.

Comment (kragniz) [inline requirements.txt:37]: I just saw I pulled osprofiler, I'll add it back in.

**Pull request merged**